src/main/java/com/google/api/codegen/nodejs/NodeJSGapicContext.java

```diff
@@ -316,6 +318,25 @@ public String getFieldType(Field field) {
   }
 
   /**
+   * Returns the JavaScript representation of the function to return the byte length.
+   */
+  public String getByteLengthFunction(TypeRef typeRef) {
+    switch (typeRef.getKind()) {
+      case TYPE_MESSAGE:
+        return "gax.createByteLengthFunction(grpcClient."
+            + typeRef.getMessageType().getFullName()
+            + ")";
+      case TYPE_STRING:
+      case TYPE_BYTES:
+        return "function(s) { return s.length; }";
+      default:
+        // It's not quite easy how to compute the byte length for numeric
+        // values. Returns '8' (necessary for 64bit integer) always for now.
+        return "function(n) { return 8; }";
```

Comment:
This doesn't seem like a good permanent solution (what about 32-bit ints?), although we don't have any examples of bundled numeric fields at the moment. Let's add a TODO here at the minimum.

Optional: just throw an exception here, which will force us to implement this later if it ever comes up in an API.